packages/terra-modal/lib/Modal.scss

```diff
@@ -0,0 +1,31 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: #fff;
+  border-radius: 5px;
+  bottom: 10px;
+  display: block;
+  left: 10px;
+  max-height: 100%;
+  outline: none;
+  overflow-x: hidden;
+  overflow-y: auto;
```

Comment:
![screen shot 2017-04-17 at 1 28 26 pm](https://cloud.githubusercontent.com/assets/8146159/25099423/ced8f44a-2371-11e7-9cc9-cb0ef084b0a7.png)

I am on the fence if we should include this for the modal. If a HOC modal dialog does not exist, this will enable overflow to scroll. However, if a HOC does exist, there is the possibility of having 2 scroll bars.